modules/router-store/src/serializers/full_serializer.ts

```diff
@@ -6,7 +6,10 @@ export interface SerializedRouterStateSnapshot extends BaseRouterStoreState {
   url: string;
 }
 
-export class DefaultRouterStateSerializer
+/**
+ * @deprecated The FullRouterStateSerializer cannot be used when serializability runtime checks are enabled. Use the {@link MinimalRouterStateSerializer} or a {@link https://ngrx.io/guide/router-store/configuration#custom-router-state-serializer custom serializer} instead.
```

Comment:
I'm not sure we need to deprecate this yet. Deprecation means this could be removed at some point. It's still a valid serializer given its limitations. At some point there was a discussion about making the router state serializable.